### PR TITLE
[#80] Merge the hydroshare 1.14.1 branch.

### PR DESCRIPTION
**Pull request by dsummersl**
I'd suggest the following method of testing this:

```bash
# stop and remove whatever docker setup you currently have:
docker-compose stop
docker-compose rm

# rebuild the docker images and the docker database from the repository:
./hsctl rebuild --db
```
Then go to http://localhost:8000 and try out all the common areas of the site.

### CODE REVIEW

**Comment by cbcunc** [comment]
Alternatively, you can
`./hsctl stop`
and either of
`scripts\cleandocker`
or
`./hsctl reset_all`

File: config/hydroshare-config.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ### Local Configuration Variables ###
-HS_PATH: /home/hydro/hydroshare
-HOST_SSL_DIR: /home/hydro/hs-certs
+HS_PATH: /home/dsummers/rs/CK/hydroshare/MyHPOM
+HOST_SSL_DIR: /home/dsummers/rs/CK/hydroshare/hs-certs
 HS_DATABASE: pg.myhpomdevelopment.sql
```
Comment (philtuskha) [inline config/hydroshare-config.yaml]: Looks like some of your personal settings for your purely Linux environment got encluded.

**Review by philtuskha — CHANGES_REQUESTED** [review]
Besides those local setup changes, this all looks good to my bleary eyes and seems to work fine.

File: config/hydroshare-config.yaml (hunk 1 above)
Comment (dsummersl) [reply config/hydroshare-config.yaml]: ooh, good catch, thanks...

**Review by philtuskha — APPROVED** [review]
👍

**Comment by cbcunc** [comment]
Revert changes to hydroshare/local_settings.py

**Pull request merged**